test/src/test/java/org/corfudb/integration/LogReplicationIT.java

```diff
@@ -1046,6 +1051,145 @@ private void testSnapshotSyncAndLogEntrySync(int numCyclesToDelayApply, boolean
         verifyPersistedLogEntryMetadata();
     }
 
+     /**
+     * Test Log Entry (delta) Sync for the case where the ACKs are arbitrarily dropped
+     * for a fixed number of times at the Source. This will test that LR is
+     * (i) not impacted by dropped ACKs
+     * (ii) Source resends msgs for which it hasn't received the ACKs
+     * (iii) Sink handles the already seen and processed msgs.
+     **/
+    @Test
+    public void testLogEntrySyncWithAckDrops() throws Exception {
```

Comment:
I see this and testLogEntrySyncWithFSMChangeAndWithAckDrop failing on this PR, there might be something that is off (perhaps in the test itself).

Reply:
yeah, this one and the testSnapshotSyncWithAckDrops are taking turns to fail.
but now when I added some print statements, the tests passed!!